libs/ui/src/ui_strings/number_strings.ts

```diff
@@ -10,19 +10,22 @@ import { Dictionary, UiStringTranslations } from '@votingworks/types';
 export const MAXIMUM_SUPPORTED_NUMBER_FOR_TTS = 45;
 export const NUMBER_STRINGS_BASE_I18N_KEY = 'number';
 
+export function getI18nKeyForNumber(value: number): string {
+  return `${NUMBER_STRINGS_BASE_I18N_KEY}${value}`;
```

Comment:
> Flatten number string keys for VxDesign compatibility

Ah yes that's my bad! I assumed the two levels of nesting for all files but `appStrings.json`

Reply:
No worries -- thought of making the change in vxdesign instead, but decided I kind of like the simplicity of keeping the `appStrings.json` flat, so considering it a feature now 😁